asignar.py

```
@@ -175,172 +175,203 @@ def graba_log(filename, data):
                                 filename=f"sorteo/{socio}",
-                                data="El socio está en el rango de edad para  %s\n"
-                                % interes,
+                                data="El socio no está inscrito ya en  %s\n" % interes,
                             )
+                            anyo = mis_socios[socio]["nacim"]
 
                             if (
-                                actividades[interes]["horario"]
-                                not in horarios_por_socio[socio]
+                                anyo >= actividades[interes]["edatMin"]
+                                and anyo <= actividades[interes]["edatMax"]
```

Pylint (3.3.7)

[refactor] Simplify chained comparison between the operands

(R1716)